From c44dad34f0eb85bcc298b350a4dc2a12222a1d2a Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 13 Mar 2014 08:08:41 +0100
Subject: [PATCH] pick up any display device, not only vga

---
 OvmfPkg/Library/PlatformBootManagerLib/BdsPlatform.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OvmfPkg/Library/PlatformBootManagerLib/BdsPlatform.c b/OvmfPkg/Library/PlatformBootManagerLib/BdsPlatform.c
index da4cb72..3381d45 100644
--- a/OvmfPkg/Library/PlatformBootManagerLib/BdsPlatform.c
+++ b/OvmfPkg/Library/PlatformBootManagerLib/BdsPlatform.c
@@ -740,7 +740,7 @@ DetectAndPreparePlatformPciDevicePath (
   //
   // Here we decide which VGA device to enable in PCI bus
   //
-  if (IS_PCI_VGA (Pci)) {
+  if (IS_PCI_DISPLAY (Pci)) {
     //
     // Add them to ConOut.
     //
-- 
1.8.3.1

