From 34626780a6ddaa2fb950293be596568d00c1f73a Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Tue, 20 Aug 2013 14:00:55 +0200
Subject: [PATCH] OvmfPkg: don't lock lock umb when running csm

---
 OvmfPkg/Csm/LegacyBiosDxe/LegacyBootSupport.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/OvmfPkg/Csm/LegacyBiosDxe/LegacyBootSupport.c b/OvmfPkg/Csm/LegacyBiosDxe/LegacyBootSupport.c
index cd4cd24f4246..a38698717221 100644
--- a/OvmfPkg/Csm/LegacyBiosDxe/LegacyBootSupport.c
+++ b/OvmfPkg/Csm/LegacyBiosDxe/LegacyBootSupport.c
@@ -1234,8 +1234,8 @@ GenericLegacyBoot (
   //
   Private->LegacyRegion->Lock (
                            Private->LegacyRegion,
-                           0xc0000,
-                           0x40000,
+                           0xe0000,
+                           0x10000,
                            &Granularity
                            );
 
-- 
2.18.1

